### PR TITLE
remove external-sql-console

### PR DESCRIPTION
**Pull request by LucyLu8L**
I've remove the external sql console page--once again. pls review

### CODE REVIEW

**Comment by tianzhou** [comment]
@LucyLu8L please take a look at the cla-bot message

**Comment by LucyLu8L** [comment]
> @LucyLu8L please take a look at the cla-bot message

I've already done this. Don't know why it reports this.

**Comment by tianzhou** [comment]
@LucyLu8L need to remove the item from TOC as well https://github.com/bytebase/bytebase.com/blob/main/content/docs/en/_layout.md#link-external-sql-console

**Review by tianzhou — CHANGES_REQUESTED** [review]
Need to remove from TOC as well https://github.com/bytebase/bytebase.com/blob/main/content/docs/en/_layout.md#link-external-sql-console

**Comment by LucyLu8L** [comment]
> Need to remove from TOC as well https://github.com/bytebase/bytebase.com/blob/main/content/docs/en/_layout.md#link-external-sql-console

oh I see, I'll close this and commit again.

File: content/docs/en/_layout.md [hunk 1]
```diff
@@ -100,8 +100,6 @@
 
 ### [Customize the Workspace with Your Own Logo](/settings/customize-workspace/)
 
-### [Link External SQL Console](/settings/external-sql-console)
```
Comment (tianzhou) [inline content/docs/en/_layout.md:103]: fooo

**Review by tianzhou — CHANGES_REQUESTED** [review]
ok

**Comment by tianzhou** [comment]
PTAL

**Review by tianzhou — APPROVED** [review]
LGTM. Epic things start with a humble step.

**Pull request merged**